### PR TITLE
Add chat footer

### LINKED ISSUE #18: Add styles to section messages

### PR DESCRIPTION
**Pull request by paulotijero**
It built the chat footer, also it was added the styles
Resolve #18

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -209,20 +215,23 @@
...
-        <svg class="-normal"><use xlink:href="#plus"></svg>
+        <div class="container-footer">
+          <svg class="-effect"><use xlink:href="#plus"></svg>
           <div class="write-message">
             <form action="" id="sendMessageForm" >
               <input
                 type="text"
                 id="txtMessage"
-                placeholder="test message"
+                placeholder="send message to #general"
```
Comment (Linzeur) [inline public/index.html]: Good detail!

File: public/assets/css/style.css [hunk 2]
```diff
@@ -372,6 +359,33 @@ body {
 .-effect:hover {
   cursor: pointer;
   fill: #fff;
-  width: 24px;
-  height: 24px;
+  transform: scale(1.2);
```
Comment (Gatts1) [inline public/assets/css/style.css]: Nice appearance for buttons

File: public/index.html [hunk 3]
```diff
@@ -50,6 +50,12 @@
         <symbol id="gif" viewBox="0 0 24 18">
           <path d="M3,0 L21,0 C22.6568542,-3.04359188e-16 24,1.34314575 24,3 L24,15 C24,16.6568542 22.6568542,18 21,18 L3,18 C1.34314575,18 2.02906125e-16,16.6568542 0,15 L0,3 C-2.02906125e-16,1.34314575 1.34314575,-5.83819232e-16 3,-8.8817842e-16 Z M11,5 L11,13 L13,13 L13,5 L11,5 Z M8,5 L4,5 C3.28,5 3,5.33333333 3,6 L3,12 C3,12.6666667 3.28,13 4,13 L8,13 C8.72,13 9,12.6666667 9,12 L9,8 L6,8 L6,10 L7,10 L7,11 L5,11 L5,7 L9,7 L9,6 C9,5.33333333 8.72,5 8,5 Z M21,7 L21,5 L15,5 L15,13 L17,13 L17,10 L20,10 L20,8 L17,8 L17,7 L21,7 Z"></path>
         </symbol>
+        <symbol id="smile" viewBox="0 0 16 16">
+          <path d="M8 1c3.9 0 7 3.1 7 7s-3.1 7-7 7-7-3.1-7-7 3.1-7 7-7zM8 0c-4.4 0-8 3.6-8 8s3.6 8 8 8 8-3.6 8-8-3.6-8-8-8v0z"/>
+          <path d="M8 13.2c-2 0-3.8-1.2-4.6-3.1l0.9-0.4c0.6 1.5 2.1 2.4 3.7 2.4s3.1-1 3.7-2.4l0.9 0.4c-0.8 2-2.6 3.1-4.6 3.1z"/>
+          <path d="M7 6c0 0.552-0.448 1-1 1s-1-0.448-1-1c0-0.552 0.448-1 1-1s1 0.448 1 1z"/>
+          <path d="M11 6c0 0.552-0.448 1-1 1s-1-0.448-1-1c0-0.552 0.448-1 1-1s1 0.448 1 1z"/>
```
Comment (Gatts1) [inline public/index.html]: Why are there so many path tags? What do they do?

Comment (paulotijero) [reply public/index.html]: Imagine `viewBox="0 0 16 16"` how to an paper with cordenates `starting point / finish point = (x y) / (x y) = (0 0) / (16 16)`. Then the `path` tags it's the order of paint in the `viewBox` so in this case the image is a `smile face` and contain many parts, for example I will tried to describe the `path`:
```HTML
<symbol id="smile" viewBox="0 0 24 18">
  <path d="coordinate of the face circle"></path>
  <path d="coordinate of the smile"></path>
  <path d="coordinate of the eye left"></path>
  <path d="coordinate of the eye right"></path>
</symbol>
```
**Image reference**
smile

**Pull request merged**